Configuration/DataProcessing/python/Impl/trackingOnlyEra_Run3.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+"""
+_trackingOnlyEra_Run3
+
+Scenario supporting proton collisions and tracking only reconstruction for HP beamspot
+
+"""
+
+import os
+import sys
+
+from   Configuration.DataProcessing.Impl.trackingOnly import trackingOnly
+import FWCore.ParameterSet.Config as cms
+from   Configuration.Eras.Era_Run3_cff import Run3
+
+from   Configuration.DataProcessing.Impl.pp import pp
+
+class trackingOnlyEra_Run3(trackingOnly):
+    def __init__(self):
+        trackingOnly.__init__(self)
+        # tracking only RECO is sufficient, to run high performance BS at PCL;
+        # some dedicated customization are required, though: customisePostEra_Run3_trackingOnly
+        self.recoSeq=':reconstruction_trackingOnly'
+        self.cbSc='pp'
+        self.addEI=False
+        self.eras=Run3
+        self.promptCustoms += [ 'Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2018' ]
+        self.expressCustoms += [ 'Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2018_express_trackingOnly' ]
+        self.visCustoms += [ 'Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2018' ]
```

Comment:
are these correct customs?
My guess is that at least the prompt and vs should stay with Run3

Reply:
ah, given there is customisePostEra_Run3_express_trackingOnly, all of the above should be Run3\*

Reply:
@slava77 this was just overlooked, fixing it